custom_dict.txt

```diff
@@ -107,6 +108,7 @@ cd
 (CF|cf)(-for-k8s)?
 classpath
 CloudEvents
+Cloudfoundry
```

Comment:
Is this not supposed to be two words - Cloud Foundry?

Reply:
I fixed that.